virttest/env_process.py

```diff
@@ -1240,6 +1240,22 @@ def preprocess(test, params, env):
             test.cancel("Got host qemu version:%s, which is not in %s" %
                         (host_qemu, required_qemu))
 
+    # Checking required virtio driver version, if not satisfied, cancel test
+    if params.get("required_virtio_win") and params.get("cdrom_virtio"):
```

Comment:
What if `required_virtio_win` defined but `cdrom_virtio` is missing, should we raise up error?

Reply:
Yes, it is a nice reminder for user.Will update it right now.
We can raise up an error like "`required_virtio_win` can not take effect without  `cdrom_virtio`".